<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_backup</class>
 <widget class="QDialog" name="Dialog_backup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>备份</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择需要备份的文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>本地备份</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打包</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>压缩</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>加密</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="backFrom">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始备份</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="backTo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(240, 240, 240);</string>
   </property>
   <property name="text">
    <string>文件路径选择不完整！</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>增量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器备份</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文件备份到的位置：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>实时</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
